pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1402,10 +1402,40 @@ func doActivateTail(ctx *domainContext, status *types.DomainStatus,
 	if err != nil {
 		log.Errorf("setupVlans failed: %v", err)
 	}
+	setupVf(ctx.assignableAdapters, status.IoAdapterList)
 	log.Functionf("doActivateTail(%v) done for %s",
 		status.UUIDandVersion, status.DisplayName)
 }
 
+func setupVf(aa *types.AssignableAdapters, ioAdapterList []types.IoAdapter) {
+	if !aa.Initialized {
+		for _, ib := range aa.IoBundleList {
+			if ib.Type == types.IoNetEthVF || ib.Type == types.IoNetEthPF {
```

Comment:
Doing this for EthVF only makes sense because the VF has the same IfName as the PF. I think that should be documented somewhere in the code.

Reply:
Commented in EthVF function